packages/frontend/electron/src/main/onboarding.ts

```diff
@@ -0,0 +1,78 @@
+import { assert } from 'console';
+import { BrowserWindow } from 'electron';
+import { join } from 'path';
+
+import { mainWindowOrigin } from './constants';
+// import { getExposedMeta } from './exposed';
+import { ensureHelperProcess } from './helper-process';
+import { logger } from './logger';
+
+// todo: not all window need all of the exposed meta
+const getWindowAdditionalArguments = async () => {
+  const { getExposedMeta } = await import('./exposed');
```

Comment:
why import in function?

Reply:
To avoid circular dependency, see https://github.com/toeverything/AFFiNE/pull/5251